runescape-api/src/main/java/net/runelite/rs/api/RSAbstractArchive.java

```diff
@@ -1,14 +1,31 @@
 package net.runelite.rs.api;
 
+import net.runelite.api.AbstractArchive;
 import net.runelite.api.IndexDataBase;
 import net.runelite.mapping.Import;
 
-public interface RSAbstractArchive extends IndexDataBase
+public interface RSAbstractArchive extends IndexDataBase, AbstractArchive
 {
 	@Import("takeFile")
 	byte[] getConfigData(int archiveId, int fileId);
 
 	@Import("getGroupFileIds")
 	@Override
 	int[] getFileIds(int group);
+
+	@Import("groupCount")
```

Comment:
You need to override where possible in rs-api

Reply:
Thanks, added those.